app/org/maproulette/models/dal/TaskDAL.scala

```diff
@@ -1144,7 +1144,7 @@ class TaskDAL @Inject() (
           s"""NOT (tasks.status != ${Task.STATUS_CREATED} AND tasks.id IN (
              |SELECT task_id FROM status_actions
              |WHERE osm_user_id IN (${user.osmProfile.id})
-             |  AND created >= NOW() - '1 hour'::INTERVAL))""".stripMargin
```

Comment:
I think there will be unintended consequences by using `modified` vs `created`. The backend has scheduled tasks that run at specific intervals and will update projects/challenges/tasks, and when a task is updated by the automation there is a trigger (`update_tasks()`) which will update the modify time of the task. The `modified` update happens **without user interaction** and is updated any time a scheduled tasks updates a task.

We could have a case where a scheduled task runs, updating the modified date, and those ready-to-be-edited-tasks are excluded from the results until the modified time is over 1 hour. If timing is really unlucky for the scheduled tasks, those could perpetually update the modified time within the hour and, essentially, always filter out all tasks.

Reply:
I'll just add something custom for the too hard status that updates the created column in its normal workflow

Reply:
Updating the `created` date would cause another set of issues, some other solution is needed.